types/plist/plist-tests.ts

```diff
@@ -0,0 +1,81 @@
+import * as fs from "fs";
+import plist = require("plist");
```

Comment:
Since the types use Es6 exports, the import should use Es6 syntax: `import * as plist from "plist"`